scripts/build_vp_ner_lm.sh

```diff
@@ -0,0 +1 @@
+bash scripts/create_ngram.sh $1 manifest/slue-voxpopuli/fine-tune.wrd save/kenlm/vp_ner 4
```

Comment:
@fwu-asapp 
I believe the second argument should be `manifest/slue-voxpopuli/e2e_ner/fine-tune.raw.wrd`

Reply:
@fwu-asapp Can you update this? so that I can merge.

Reply:
@sshon-asapp I've fixed this.